docs/resources/Guild.md

```diff
@@ -720,16 +720,20 @@ Create a new [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object for the gu
 | position                      | integer                                                                        | sorting position of the channel                                                                                                                                                 |
-| permission_overwrites\*       | array of partial [overwrite](#DOCS_RESOURCES_CHANNEL/overwrite-object) objects | the channel's permission overwrites                                                                                                                                             |
+| permission_overwrites\*\*     | array of partial [overwrite](#DOCS_RESOURCES_CHANNEL/overwrite-object) objects | the channel's permission overwrites                                                                                                                                             |
 | parent_id                     | snowflake                                                                      | id of the parent category for a channel                                                                                                                                         |
 | nsfw                          | boolean                                                                        | whether the channel is nsfw                                                                                                                                                     |
+| rtc_region                    | string                                                                         | channel [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) id of the voice or stage channel, automatic when set to null                                                  |
+| video_quality_mode            | integer                                                                        | the camera [video quality mode](#DOCS_RESOURCES_CHANNEL/channel-object-video-quality-modes) of the voice channel                                                                |
 | default_auto_archive_duration | integer                                                                        | the default duration that the clients use (not the API) for newly created threads in the channel, in minutes, to automatically archive the thread after recent activity         |
 
-\* In each overwrite object, the `allow` and `deny` keys can be omitted or set to `null`, which both default to `"0"`.
+\* For voice channels, normal servers can set bitrate up to 96000, servers with Boost level 1 can set up to 128000, servers with Boost level 2 can set up to 256000, and servers with Boost level 3 or the `VIP_REGIONS` [guild feature](#DOCS_RESOURCES_GUILD/guild-object-guild-features) can set up to 384000. For stage channels, bitrate can be set up to 64000.
```

Comment:
Feel free to take what you want from this, but I just tried to clean it up a bit since it's a long note. Not sure about bolding voice channels/stage channels, but stage channels feels buried so that's reasoning.

```suggestion
\* For **voice channels**, `bitrate` can be set up to `96000` in non-boosted servers. In [boosted](https://support.discord.com/hc/en-us/articles/360028038352-Server-Boosting-FAQ-#h_419c3bd5-addd-4989-b7cf-c7957ef92583) servers, it can be set up to `128000` for level 1, up to `256000` for level 2, and up to `384000` for level 3 or in servers with the `VIP_REGIONS` [guild feature](#DOCS_RESOURCES_GUILD/guild-object-guild-features). For **stage channels**, `bitrate` can be set up to `64000`.
```